### PR TITLE
Structure yarn aliases

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
## Description

Add new several `yarn` aliases where each alias consists of first two letters of each word.

## Motivation and Context

The goal is to simplify alias memorizing without breaking backward compatibility.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] If I have added a new file, I also added it to ``clean_files.txt`` and formatted it using ``lint_clean_files.sh``.
- [ ] I have added tests to cover my changes, and all the new and existing tests pass.

### CODE REVIEW

**Comment by EmilySeville7cfg** [comment]
P.S. Sorry for this question but I don't completely understand how can I make Bash-it to use my local repo version with these new aliases.

**Comment by gaelicWizard** [comment]
> P.S. Sorry for this question but I don't completely understand how can I make Bash-it to use my local repo version with these new aliases.

@EmilySeville7cfg, easiest thing unless you expect to make a lot of personal modifications is to just copy the file under `custom` and it will get loaded. 

Proper answer to your question: change your git remotes to point to your fork as "origin" and add the main repo as "upstream". 

If you don't intend to make a whole bunch of changes, I would opt for the former. It's somewhat painful rebasing so frequently (as you can see I have so many open PRs).

**Comment by EmilySeville7cfg** [comment]
It is said:

> When adding new files, be sure to add them into clean_files.txt, which is a growing list of linted files in the project.

But I don't see all alias files in `clean_files.txt`. 🤔 

```
# aliases
#
aliases/available/dnf.aliases.bash
aliases/available/git.aliases.bash
aliases/available/vim.aliases.bash
```

I am not sure whether it's an error or an intended decision.

**Comment by gaelicWizard** [comment]
Most files aren't under `clean_files.txt` due to simply nobody putting in the work to do so. Most of my open PRs are specifically doing the work to get them in there. #2047 adds the entire `aliases/` folder to `clean_files.txt`, but its awaiting merge.

**Comment by NoahGorny** [comment]
Hi @EmilySeville7cfg, this seems useful! Thanks for contributing 😄 
I do not use `yarn` at all, so I dont have strong feelings about the terminology.
@davidpfarrell, do you have any strong feelings about this one?

**Comment by gaelicWizard** [comment]
@EmilySeville7cfg, no longer any need to add to `clean_files.txt`: the whole `aliases` folder is now added! 🎉

**Review by davidpfarrell — CHANGES_REQUESTED** [review]
Greetings,

Looks like duplicate entries weren't deleted (see inline comment)

NOTE: You have `yain` twice ("yarn init" and "yarn info")

In general, I am *not* a fan of just blindly using first-n letters to create aliases.

You want aliases to be 
* Short
* Memorable
* Generally Easy To Type
* Unambiguous
* Actions Used Most Often Are Easiest To Type

I think achieving these is more nuanced than a flat first-n pattern can offer.

I don't know if all aliases need to start with `ya` - Not sure if there's that much contention for y* aliases?

Short words might be better just spelled than tried to make shorter.

Here's my take on how I might define these aliases:

ya

yinit

yadd
ygladd | ygadd
ydevadd | ydvadd

yup

yrm
rglrm | ygrm
ydevrm | ydvrm

yout
ypack
ypub
ysup

yrun
ytest
yserve

yclean
ycheck
ylist | yls
yinfo

ylslic | ylistlic # Maybe just remove this - No good alias for it and likely not used often?

TL;DR

I don't use Yarn much so I'm not invested in these specific aliases, but in general I'd prefer that Bash-it be a reflection of existing aliases in-use by the community and less trying to pioneer new ground. 

Also, once we introduce them,  we're going to be reluctant to remove them later.
